cogs/helpers.py

```diff
@@ -510,6 +510,31 @@ def __init__(self, bot):
         except ValueError:
             ctx.send("Input must be integers")
 
+    @commands.command()
+    async def food(self, ctx, *args):
+        """Posts a food sale in #foodspotting.
+        Use: `?food Samosas in leacock`
+        You can also attach one picture to your message"""
```

Comment:
Does the command have to be added in the uploaded image caption or can the image be added in the next message after the command?

Reply:
This should be specified in the help docstring.

Reply:
The command has to be added in the uploaded image caption